python/ray/dashboard/state_aggregator.py

```diff
@@ -181,6 +181,7 @@ def _filter(
                 if filter_column not in filterable_columns:
                     raise ValueError(
                         f"The given filter column {filter_column} is not supported. "
+                        "Enter filters with key=val or key!=val. "
```

Comment:
While you are here, can you fix the type annotations for `_filter` and `_convert_filters_type`, the type for `filters` should be `List[Tuple[str, PredicateType, SupportedFilterType]]`

Reply:
Thanks. Looks like your comments only contains the type changes and missing other changes.

Reply:
Ya so i removed the key check bc of jiajun's comment, think it's overkill to check if key on filter is alphabetical and the new line in the message should give the user enough direction to figure out what's wrong